explainability-core/src/test/java/org/kie/trustyai/metrics/drift/fouriermmd/FourierMMDTest.java

```diff
@@ -138,4 +182,55 @@ void testProductionData() {
             e.printStackTrace();
         }
     }
+
+    @Test
+    void testRandomData() {
+        try {
```

Comment:
this `try` block is unnecessary, you just need `testRandomData throw IOException`

Reply:
fixed